### DISCUSSION TITLE
Unable to install Encoder

### DISCUSSION THREAD

**Discussion by gregfox151** [micropython/micropython#13300] [ESP32]
Hi, I need encoder in my program, and when I attempt to install it  it warns it is for Raspberry pi .
Does anyone know of an option

**Comment by fdufnews** [comment]
Maybe you can give a little more information concerning the product you are using.
Put a link to the site you bought the encoder and a little drawing of how it was wired.

**Reply by jken12** [reply]
Hi[,](https://glassworkindubai.com/) please also help me regarding this. How we can interlink this?

**Reply by robert-hh** [reply]
You could have a looks at the encoder code of @peterhinch : https://github.com/peterhinch/micropython-samples/tree/master/encoders
They are portable.
